internal/services/firewall/firewall_data_source.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/hashicorp/go-azure-helpers/resourcemanager/tags"
 	"github.com/hashicorp/go-azure-helpers/resourcemanager/zones"
 	"github.com/hashicorp/go-azure-sdk/resource-manager/network/2024-05-01/azurefirewalls"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```

Comment:
This package shouldn't be directly used, use the wrapped types in `github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk` instead.